packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -123,98 +123,129 @@ class BouncingScrollSimulation extends Simulation {
+/// of the current velocity [dx] and does not depend on how long ago the
+/// simulation was started.  (This is required by Flutter's scrolling protocol,
+/// where [ScrollActivityDelegate.goBallistic] may restart a scroll activity
+/// using only its current velocity and the scroll position's own state.)
+/// Compared to this scroll physics, Android's moves faster at the very
+/// beginning, then slower, and it ends at the same place but a little later.
+///
+/// Times are measured in seconds, and positions in logical pixels.
 ///
 /// See also:
 ///
 ///  * [BouncingScrollSimulation], which implements iOS scroll physics.
 //
-// This class is based on Scroller.java from Android:
-//   https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/widget
+// This class is based on OverScroller.java from Android:
+//   https://android.googlesource.com/platform/frameworks/base/+/android-13.0.0_r24/core/java/android/widget/OverScroller.java#738
+// and in particular class SplineOverScroller (at the end of the file), starting
+// at method "fling".  (A very similar algorithm is in Scroller.java in the same
+// directory, but OverScroller is what's used by RecyclerView.)
 //
-// The "See..." comments below refer to Scroller methods and values. Some
-// simplifications have been made.
+// In the Android implementation, times are in milliseconds, positions are in
+// physical pixels, but velocity is in physical pixels per whole second.
+//
+// The "See..." comments below refer to SplineOverScroller methods and values.
 class ClampingScrollSimulation extends Simulation {
-  /// Creates a scroll physics simulation that matches Android scrolling.
+  /// Creates a scroll physics simulation that aligns with Android scrolling.
   ClampingScrollSimulation({
     required this.position,
     required this.velocity,
     this.friction = 0.015,
     super.tolerance,
-  }) : assert(_flingVelocityPenetration(0.0) == _initialVelocityPenetration) {
-    _duration = _flingDuration(velocity);
-    _distance = (velocity * _duration / _initialVelocityPenetration).abs();
+  }) {
+    _duration = _flingDuration();
+    _distance = _flingDistance();
   }
 
-  /// The position of the particle at the beginning of the simulation.
+  /// The position of the particle at the beginning of the simulation, in
+  /// logical pixels.
   final double position;
 
   /// The velocity at which the particle is traveling at the beginning of the
-  /// simulation.
+  /// simulation, in logical pixels per second.
   final double velocity;
 
   /// The amount of friction the particle experiences as it travels.
   ///
-  /// The more friction the particle experiences, the sooner it stops.
+  /// The more friction the particle experiences, the sooner it stops and the
+  /// less far it travels.
+  ///
+  /// The default value causes the particle to travel the same total distance
+  /// as in the Android scroll physics.
+  // See mFlingFriction.
```

Comment:
This should take the form of a `See also` and a breadcrumb `[]` like in other docs.

Reply:
Cool. Yeah, this is a reference to the corresponding Android source (like some other private comments in this class), so I think it's useful only for someone actually reading the implementation.